rosdep/base.yaml

```diff
@@ -4222,6 +4223,7 @@ phantomjs:
   debian:
     buster: [phantomjs]
     stretch: [phantomjs]
+  fedora: [python-selenium]
```

Comment:
This looks like a copy and paste error? I don't see documentation that python-selenium includes/provides phantomjs.

Reply:
https://apps.fedoraproject.org/packages/s/phantomjs indicates seleniumn provides phantomjs

Reply:
The search was getting a hit from the description of the `python-selenium` package:
```
The selenium package is used automate web browser interaction from Python.

Several browsers/drivers are supported (Firefox, Chrome, Internet Explorer,
PhantomJS), as well as the Remote protocol.
```

It doesn't appear that `phantomjs` is packaged for Fedora:
```
$ dnf provides '*bin/phantomjs'
Last metadata expiration check: 0:00:54 ago on Fri 22 Mar 2019 09:38:26 AM PDT.
Error: No Matches found
```

Reply:
thanks for the comment, I removed fedroa section